Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -712,8 +712,17 @@ void MainWindow::loadSettings()
             this->ui.retranslateUi(this);
         else
         {
-            QMessageBox::information(this, tr("Translate"),
-                QString("Failed to load language file %1").arg(lang));
+            QString langPrefix = lang.section('_', 0, 0);
+            if (switchLanguage(langPrefix))

```

Comment:
I think I've asked this before. But doesn't this change to "langPrefix" strip "_BR" and "_PT" from Portuguese language file?

Reply:
Ah yes, when you asked previously it was not the case in the previous context but you're right for here. I'll look into it tomorrow